lessons/3_first_interactive_component/index.html

```diff
@@ -331,29 +364,92 @@ <h3>For that you need to do:</h3>
 					</li>
 				</ul>
 			</section>
+			<section>
+				<h3>But where do we write JavaScipt code?</h3>
+			</section>
+			<section>
+				<p>We will use <b>&lt;script/&gt;</b> tag inside <b>index.html</b></p>
+				<pre>
+					<code>
+&lt;html&gt;
+ &lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+	&lt;title&gt;My HTML Page&lt;/title&gt;
+ &lt;/head&gt;
+ &lt;body&gt;
+  ...
+  &lt;script type="text/javascript"&gt;
+  ...
+  &lt;/script&gt;
+ &lt;/body&gt;
+&lt;/html&gt;
+					</code>
+				</pre>
+			</section>
+			<section>
+				<h3>Example with buttons</h3>
+				<pre>
+					<code>
+&lt;html&gt;
+ &lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+	&lt;title&gt;My HTML Page&lt;/title&gt;
+ &lt;/head&gt;
+ &lt;body&gt;
+  &lt;p&gt;How are you?&lt;/p&gt;
+  &lt;button id="goodButton"&gt;Good 👍&lt;/button&gt;&lt;button id="badButton"&gt;Bad 👎&lt;/button&gt;
+  &lt;p id="answer">&lt;/p&gt;
+	&lt;script&gt;
+	 let goodButtonCmp = document.getElementById("goodButton");
+	 let badButtonCmp = document.getElementById("badButton");
+	 let answerCmp = document.getElementById("answer");
+
+	 function goodButtonClick(event) {
+		answerCmp.textContent = "That's great 😊";
+	 }
+
+ 	 function badButtonClick(event) {
+		answerCmp.textContent = "Oh that's sad 😥";
+	 }
+
+	 goodButtonCmp.addEventListener("click", goodButtonClick);
+	 badButtonCmp.addEventListener("click", badButtonClick);
+	&lt;/script&gt;
+ &lt;/body&gt;
+&lt;/html&gt;
+					</code>
+				</pre>
+			</section>
 			<section>
 				<h3>Now let's practice</h3>
 			</section>
 			<section>
 				<h3>Please do the following</h3>
 				<ul>
-					<li>Create a separate page in main project</li>
-					<li>Create a button</li>
-					<li>Make this button do the console.log('Clicked') if user clicked it</li>
+					<li>Copy your index.html file and name it Lesson3.html</li>
+					<li>Create a Feel good button </li>
+					<li>Make this button do the console.log('I feel really good today') if user clicked it</li>
 					<li>HINT: for that you need to apply all what we just learned :)</li>
 				</ul>
 			</section>
 			<section>
 				<h1>Homework</h1>
```

Comment:
I think we should move all homework to markdown files that clean easy to reference place for the students like we did for homework one https://mrtim.github.io/js-munich-2020-spring/homework/1_base_page

Reply:
Normally there is a separate slides set for homework, like here https://mrtim.github.io/js-munich-2020-spring/lessons/2_git_time/homework#/3
I just forgot to make the same thing here.